runtime/src/runtime.lisp

```diff
@@ -111,13 +123,20 @@
         (format log-stream "starting bot ~a using command ~a~%" 
                 (name bot-definition)
                 memory-limited-command)
-        (let ((bot (make-instance 
-                    'concrete-bot
-                    :bot-process (run-bot "bash" (list "-c" memory-limited-command))
-                    :bot-id (random-id 10)
-                    :bot-name (name bot-definition)
-                    :bot-definition bot-definition
-                    :command memory-limited-command)))
+        (let* ((bot-id (random-id 10))
+               (bot-name (name bot-definition))
+               (bot (make-instance
+                     'concrete-bot
+                     :bot-process (run-bot "bash"
+                                           (list "-c" memory-limited-command)
+                                           (open (format nil "~a-~a.err" bot-name bot-id)
```

Comment:
Same here. I suggest an additional argument to this function. I'm happy to contribute to this change if you'd like.

Reply:
Nope.  I'm happy to do that myself :)